sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/CqlDataWriterDAO.java

```diff
@@ -195,8 +196,17 @@ private Statement deleteStatement(TableDDL tableDDL, ByteBuffer rowKey, UUID cha
                 .setConsistencyLevel(consistencyLevel);
     }
 
+    private Statement blockedDeleteStatement(BlockedDeltaTableDDL tableDDL, ByteBuffer rowKey, UUID changeId, int block, ConsistencyLevel consistencyLevel) {
```

Comment:
Can we add a comment above saying this will create cell tombstones instead of range?

Reply:
done